model/client.go

```diff
@@ -40,13 +40,19 @@ func (c *Client) CreateTranslation(translationRequest *TranslationRequest) (*Tra
 	if err != nil {
 		return nil, err
 	}
-	defer closeBody(resp)
+
+	bodyBytes, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, errors.New("failed to read response body")
+	}
+	defer resp.Body.Close()
```

Comment:
We need to defer `Close()` before reading body. Also I think `Close()` is not enough. We need `io.Copy(io.Discard, resp.Body)` too.

Reply:
If we don't use the body, we may need to discard the data. But since we `io.Readall` that is not needed. I will take a look at it though, maybe we can improve the workflow somehow.